python/python/glide/async_commands/server_modules/ft.py

```diff
@@ -33,7 +33,7 @@ async def create(
         options (Optional[FtCreateOptions]): Optional arguments for the [FT.CREATE] command.
 
     Returns:
-        If the index is successfully created, returns "OK".
+        Returns a simple string "OK" message or an error reply.
```

Comment:
```suggestion
        A simple "OK" response.
```

might be better, like the rest of the ok commands

Reply:
I have copied this exactly from the vector search documentation 
but I can change this for consistency with other commands. Thanks.

Reply:
Updated in the next commit.